PinpointKit/PinpointKit/Sources/Core/Editing/EditorDelegate.swift

```diff
@@ -26,11 +34,28 @@ public protocol EditorDelegate: class {
     - returns: A bool value that defines if the editor dismisses or not.
      */
     func editorShouldDismiss(_ editor: Editor, with screenshot: UIImage) -> Bool
+    
+    /**
+     A function that is called any time the editor makes a modification to the screenshot.
+     
+     - parameter editor: The editor resonsible for editing the image.
+     - parameter screenshot: The edited image of a screenshot.
+     */
+    func editorDidMakeChanges(_ editor: Editor, to screenshot: UIImage)
 }
 
 /// Extends editor delegate with base implementation for functions.
 extension EditorDelegate {
+    
+    public func editorDidDismiss(_ editor: Editor, with screenshot: UIImage) {
+        // Do nothing
+    }
+    
     public func editorShouldDismiss(_ editor: Editor, with screenshot: UIImage) -> Bool {
         return true
     }
+    
+    public func editorDidMakeChanges(_ editor: Editor, to screenshot: UIImage) {
```

Comment:
I think we should order this above the dismissal methods, so as to suggest the typical chronology.